docs/Rakefile

```diff
@@ -33,7 +33,7 @@ task :clean do
   rm_f ADOC_VARIABLES
   # find ditaa images based on the fact that they have a cache file
   DITAA_IMAGES = Rake::FileList.new do |list|
-    Rake::FileList.new("source/**/images/*.png.cache").each do |f|
+    Rake::FileList.new("source/**/images/*-template.png.cache").each do |f|
```

Comment:
@budhrg `source/**/images/*-template.png.cache` what is the use of this, are you forgot to `cp` or `mv` to `minishift-architecture-template.png` or `minishift-architecture.png` ?